pkg/configurator/client.go

```diff
@@ -165,7 +165,8 @@ func getEgressCIDR(configMap *v1.ConfigMap) string {
 func getBoolValueForKey(configMap *v1.ConfigMap, key string) bool {
 	configMapStringValue, ok := configMap.Data[key]
 	if !ok {
-		log.Debug().Err(errInvalidKeyInConfigMap).Msgf("Key %s does not exist in ConfigMap %s/%s (%s)", key, configMap.Namespace, configMap.Name, configMap.Data)
+		log.Debug().Msgf("Key %s does not exist in ConfigMap %s/%s (%s)",
```

Comment:
with the deletion of the error, `errInvalidKeyInConfigMap` won't be needed any more, please remove the reference from `errors.go`